languages/rails.md

```diff
@@ -53,25 +53,26 @@ SimpleCov.start('rails')
 - Commit on a new branch and push your changes
 
 - Open a PR and your Action should run (and pass!). If not, debug or ask on our
-Slack for tips.
+  Slack for tips.
 
 ## Rails Development Environment
 
+- For local development, you will need the ENV for the app you're working with.
+  Contact an engineer who's worked on the app, and they will provide you with
+  the requisite .env file
 - We maintain a [VagrantFile](https://github.com/MITLibraries/mit_vagrant_dev)
-  to help developers join our Rails projects with minimal effort
-- It is not a requirement to use it, but if you are having (or causing)
-  problems and aren't using it, please use it and see if it helps
+  to help developers join our Rails projects with minimal effort. It is not a
```

Comment:
Yeah, let's remove this bit about the vagrant file as I think I prefer either native rbenv or docker solutions for local dev. I'm definitely not actively maintaining that vagrant solution.